File: python/one.py
# Original array
arr1 = [2, 7, 5, 9, 5, 3, 4]

# Find max value and its index (based on original array)
max_value = arr1[0]
max_index = 0

for i in range(len(arr1)):
    if max_value < arr1[i]:
        max_value = arr1[i]
        max_index = i

# Sort array in ascending order
sorted_arr = sorted(arr1)
print("Sorted Array:", sorted_arr)

# Sort array in descending order
reverse_sorted_arr = sorted(arr1, reverse=True)
print("Reversed Sorted Array:", reverse_sorted_arr)

# Print max value and index
print("Max number of the Array:", max_value)
print(f"Max number: {max_value}'s index of the Array:", max_index)

File: java/one.java
import java.util.Arrays;
import java.util.Collections;

class Main{
    public static void main(String[] args){
        Integer[] arr1={5,3,8,90,65,7,4,5,6,97,4,2,8};
        Integer max=arr1[0];
        Integer maxindex=0;
                Integer sum=0;

        
        for(Integer i=0; i<arr1.length; i++){
            if(max<arr1[i]){
                max=arr1[i];
                maxindex=i;
            }
            sum+=arr1[i];
        }
        
       System.out.println("Max num of the Array:" +max);
              System.out.println("Max num:" +max +"'s index:" +maxindex);
              
                            System.out.println("Total of the Array:" +sum);

        
        Arrays.sort(arr1);
        
        System.out.println("Sorted Array:" +Arrays.toString(arr1));
        
        
        Arrays.sort(arr1, Collections.reverseOrder());
        
        System.out.println("Reversed Sorted Array:" +Arrays.toString(arr1));
    }
}